src/applications/financial-status-report/components/otherAssets/VehicleSummaryWidget.jsx

```diff
@@ -20,10 +24,6 @@ const VehicleSummaryWidget = ({
   const { assets } = data;
   const { automobiles = [] } = assets;
 
-  useEffect(() => {
-    clearJobIndex();
-  }, []);
-
```

Comment:
Removed because I couldn't determine the relationship between job index and automobiles